### DISCUSSION TITLE
possibility for this to work with nations from mods

### DISCUSSION THREAD

**Discussion by maoouuu** [ThiaudioTT/hoi4-presence#33] [Ideas]
i was wondering if there was any possibility for this to work with nations that come from mods like TNO or KRDX as it doesnt seem to work quite well with them

**Comment by ThiaudioTT** [comment]
Well, I think it depends. What happens with the presence when you join a country that is a mod?

**Reply by maoouuu** [reply]
instead of a county name it just displays the country tag so "USSR = SOV" as well as the flag not displaying but it works perfectly fine in vanilla

**Reply by ThiaudioTT** [reply]
> "USSR = SOV" 

Is just an example, right? Because we have an entry for it (should be displaying the flag of USSR).


I think we can do for entire new countries, existing ones will be displayed as in vanilla because we need some logic to be worked on the mod (which probably we won't get it easily).

**Reply by maoouuu** [reply]
yea i was just using that tag as an example for modded country tags i can send a screenshot if you want

**Reply by ThiaudioTT** [reply]
We can do. 

Can you provide which mods are and the list of the countries with their tags?

In hoi4 vanilla, we have a [wiki](https://hoi4.paradoxwikis.com/Countries) that displays a list of countries and their tag in the game.


Since the countries are inserted manually, open an **issue** and provide the wiki in the issue (if you can't open an [issue](https://github.com/ThiaudioTT/hoi4-presence/issues), I will open myself with the info you provide here).

Example that you will do:

**Issue 1:**
Add countries from mod X
(provide wiki and an optional list of the countries)

**Issue 2:**
Add countries from mod Y
(provide wiki and an optional list of the countries)


Doing this we can work on each mod individually.
